sosw/components/test/unit/test_helpers.py

```diff
@@ -809,6 +809,18 @@ def test_is_event_from_sqs__many(self):
         self.assertEqual([{"hello": "I am Inigo Montoya"}, {"hello2": "I am Inigo Montoya2"}],
                          unwrap_event_recursively(deepcopy(SQS_EVENT_MANY)))
 
+    def test_small_int_from_string(self):
+        assert small_int_from_string("hello world") == 91
```

Comment:
Please use`self.assertEqual()` for consistency in case we inject some middleware for unit testing.

Reply:
Ok